README.md

```diff
@@ -10,6 +10,13 @@ examples get run and tailor the output.
     gem install rspec-core # for rspec-core only
     rspec --help
 
+  Want to run against the `master` branch? You'll need to include the dependent
+  RSpec repos as well. Add the following to your `Gemfile`:
+
+    %w[rspec rspec-core rspec-expectations rspec-mocks rspec-support].each do |lib|
```

Comment:
I had originally tried this in my local "test" sandbox directory and I had to have `rspec`. I just tried again with `rspec` removed from the list and deleted the `Gemfile.lock`. Things install fine. Not sure what the issue was previously, likely something out of date in the `Gemfile`. I'll update it.